scheduler/pkg/plugins/wasm/config.go

```diff
@@ -0,0 +1,8 @@
+package wasm
+
+type PluginConfig struct {
+	// GuestName is the name of the guest wasm.
+	GuestName string `json:"guestName"`
+	// GuestPath is the path to the guest wasm.
+	GuestPath string `json:"guestPath"`
```

Comment:
cool. later I will help switch this to URI based

Reply:
FYI, maybe GuestName might be optional as we can parse the .wasm file. Anyway, would not block this PR.